### DISCUSSION TITLE
Switch back to global map after spot cleaning

### DISCUSSION THREAD

**Discussion by thepigeonoftime** [Hypfer/Valetudo#1721] [Q&A / Support]
Hi, when I perform a spot cleaning (using Dreame Z10 Pro), the map changes to the cleaned spot and seems only revertible when docking the robot or initiating (and aborting) a general cleaning via power button on the robot. Is there another way to switch back to the global ( / "normal" ) map?

**Comment by Hypfer** [comment]
Can you illustrate the issue with a screenshot? I don't quite get what you mean exactly

**Comment by thepigeonoftime** [comment]
Maybe spot cleaning isn't universal, it's available via hardware button on the Z10 Pro. When used, Valetudo creates an ad-hoc map of the spot it cleaned and I don't see a way to reset it to the overall map of our flat except with the methods mentioned above. Granted it's a minor issue, I was just curious if I'm overlooking a UI element to do so. 

![signal-2023-02-26-195119_003](https://user-images.githubusercontent.com/4933693/221430928-99387711-8335-4bac-8c00-11b123710c68.png)
